src/server/server_family.cc

```diff
@@ -2373,6 +2373,7 @@ void ServerFamily::Info(CmdArgList args, const CommandContext& cmd_cntx) {
     append("rdb_save_usec", m.coordinator_stats.rdb_save_usec);
     append("rdb_save_count", m.coordinator_stats.rdb_save_count);
     append("big_value_preemptions", m.coordinator_stats.big_value_preemptions);
+    append("compressed_blobs", m.coordinator_stats.compressed_blobs);
```

Comment:
i think it belongs to "PERSISTENCE"  section below.

Reply:
actually not sure it belongs to   "PERSISTENCE" as all the stats there are related for current/last save operations. but this can be related to replication as well.